.gitignore

```diff
@@ -1,10 +1,2 @@
-# vendor tree-sitter code
-/vendor/tree-sitter/*
-!/vendor/tree-sitter/lib
-/vendor/tree-sitter/lib/*
-!/vendor/tree-sitter/lib/include
-!/vendor/tree-sitter/lib/src
-!/vendor/tree-sitter/lib/utf8proc
-/vendor/tree-sitter/lib/utf8proc/*
-!/vendor/tree-sitter/lib/utf8proc/utf8proc.h
-!/vendor/tree-sitter/lib/utf8proc/*.c
+# tree-sitter library file
+lib.c
```

Comment:
you delete this file in `vendor.sh`

Reply:
Sure, but just to make sure it's not committed by accident. It will completely break the build if it does.